Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -575,6 +579,28 @@ define([
                 this._resolutionScale = value;
                 this._forceResize = true;
             }
+        },
+
+        /**
+        * Boolean flag indicating if the browser's recommended resolution is used.
+        * If true, the browser's device pixel ratio is ignored and 1.0 is used instead,
+        * effectively rendering based on CSS pixels instead of device pixels. This can improve
+        * performance on less powerful devices that have high pixel density. When false, rendering
+        * will be in device pixels. The resolutionScale property will still take effect whether
+        * this flag is true or false.
+        * @memberof CesiumWidget.prototype
+        *
+        * @type {Boolean}
+        * @default false
+        */
+        useBrowserRecommendedResolution : {
+            get : function() {
+                return this._useBrowserRecommendedResolution;
+            },
+            set : function(value) {
+                this._useBrowserRecommendedResolution = value;
+                this._forceResize = true;
```

Comment:
Maybe only set `forceResize` if the value changed. Same comment applies to `resolutionScale` (which this was modeled after)